src/components/JoinUs/Perks.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import styled from 'styled-components'
+import remcalc from 'remcalc'
+import breakpoint from 'styled-components-breakpoint'
+import { Row, Col } from '../grid'
+import { Padding } from 'styled-components-spacing'
+import { Paragraph, H2 } from '../Typography'
+import Image from '../Common/Image'
+
+import { Section, ShiftedColumn } from './elements'
+
+const PerksListContainer = styled(ShiftedColumn)`
+  display: grid;
```

Comment:
As much as I love the idea of using CSS Grid!! :heart_eyes:, I think it's worth us deciding on common conventions to use throughout the codebase. Since we don't use CSS Grid anywhere else to achieve our columns & rows, I'm not sure that it's a great idea to be mixing solutions?

To be honest, I think it would make our lives muuuuch easier if we moved to CSS Grid everywhere, but that's a big refactor job & I think consistency is more important than _which_ solution you go with.

But I'm totally open to ideas, if you think it's worth deviating from our conventions for this scenario alone?

Reply:
imo CSS grid was the adequate solution for this, regarding the control of the spacings and horizontal/vertical alignment. I don't think it should be used in replacement of the current row/column solution, but as a complement.

Besides Grid having a great browser coverage, before using it, I searched for usages of `grid` in the project and found at list to, so I considered it as a valid solution for the current conventions.

Reply:
Fair enough.

You're probably right. Given that this is a component which requires spacing along both axes, CSS Grid is the right solution.

What I'm commenting on is probably more that [we need documentation](https://trello.com/c/DrXHIMWe/207-document-how-our-grid-works-should-be-used-in-combination-with-the-other-style-libraries-we-use) actually